tests/tests.rs

```diff
@@ -856,4 +857,46 @@ mod tests {
         assert_eq!(eif_info.is_signed, true);
         assert!(eif_info.cert_info.is_some());
     }
+
+    #[test]
+    fn get_certificate_pcr() {
+        let dir = tempdir().unwrap();
+        let dir_path = dir.path().to_str().unwrap();
+        let eif_path = format!("{}/test.eif", dir_path);
+        let cert_path = format!("{}/cert.pem", dir_path);
+        let key_path = format!("{}/key.pem", dir_path);
+        generate_signing_cert_and_key(&cert_path, &key_path);
+
+        setup_env();
+        let args = BuildEnclavesArgs {
+            docker_uri: SAMPLE_DOCKER.to_string(),
+            docker_dir: None,
+            output: eif_path,
+            signing_certificate: Some(cert_path.clone()),
+            private_key: Some(key_path),
+        };
+
+        build_from_docker(
+            &args.docker_uri,
+            &args.docker_dir,
+            &args.output,
+            &args.signing_certificate,
+            &args.private_key,
+        )
+        .expect("Docker build failed");
+
+        // Describe EIF and get PCR8
+        let eif_info = describe_eif(args.output).unwrap();
+        // Hash signing certificate and verify that PCR8 is the same (identifying the certificate)
+        let pcrs = get_file_pcr(cert_path, PcrType::SigningCertificate).unwrap();
```

Comment:
"pcr" instead of "pcrs"? The function would return a single PCR value, if I recall correctly.

Reply:
Yes, it's a map containing only one PCR.